x/auction/keeper/genesis.go

```diff
@@ -10,6 +12,12 @@ import (
 func InitGenesis(ctx sdk.Context, k Keeper, gs types.GenesisState) {
 	k.setParams(ctx, gs.Params)
 
+	auctionAccount := k.GetAuctionAccount(ctx)
+	if auctionAccount == nil {
+		panic(fmt.Sprintf("%s module account has not been set", types.ModuleName))
+	}
+	k.accountKeeper.SetModuleAccount(ctx, auctionAccount)
```

Comment:
We touched on this in the other PR, probably want to drop this.

Reply:
yup, removed.